hotkey_linux.go

```diff
@@ -135,7 +135,7 @@ const (
 
 // Key represents a key.
 // See /usr/include/X11/keysymdef.h
-type Key uint8
+type Key uint16
```

Comment:
Also, I don't know if this is necessary: Is there an issue if we use uint16 across all platforms?

Reply:
There shouldn't be an issue. For now I only use uint8 for Windows and MacOS, because they don't seem to need more than a byte to store all keycodes.